src/api/lessons/types.ts

```diff
@@ -8,6 +8,11 @@ export enum LessonType {
   Homework = 'Homework',
 }
 
+export enum WebinarStatus {
+  Planned = 0,
+  Completed = 1,
```

Comment:
```suggestion
  Planned,
  Completed ,
```

В енамах значения по умолчанию нумеруются с 0 с шагом 1